bundle/language_en.properties

```diff
@@ -1889,3 +1753,152 @@ angal.xmpp.isnowonline
 angal.xmpp.sendfile                                                              = Send File
 angal.xmpp.user                                                                  = User
 angal.xmpp.usersinfo                                                             = user's info
+
+############################
+# Key/values used in sql files
+#   not changing them at this time to simplify the task
+############################
+angal.admission.admission                                                        = Admission
+angal.admission.data                                                             = Data
+angal.admission.deletepatient                                                    = Delete patient
+angal.admission.editpatient                                                      = Edit Patient
+angal.admission.examination                                                      = Examination
+angal.admission.malnutritioncontrol                                              = Malnutrition control
+angal.admission.merge                                                            = Merge
+angal.admission.newpatient                                                       = New patient
+angal.admission.opd                                                              = OPD
+angal.admission.patientfolder                                                    = Clinical sheet
+angal.admission.therapy                                                          = Therapy
+angal.agetype.adolescents                                                        = Adolescent
+angal.agetype.adult                                                              = Adult
+angal.agetype.earlychildhood                                                     = Early Childhood
+angal.agetype.elderly                                                            = Elderly
+angal.agetype.latechildhood                                                      = Late Childhood
+angal.agetype.newborn                                                            = Newborn
+angal.billbrowser.deletebill                                                     = Delete Bill
+angal.billbrowser.editbill                                                       = Edit Bill
+angal.billbrowser.newbill                                                        = New Bill
+angal.billbrowser.report                                                         = Report
+angal.common.delete                                                              = Delete
+angal.common.deletem                                                             = DELETE
```

Comment:
I remember we decided to use toUpperCase() and to avoid `angal.common.<key>m` duplicates

Reply:
Please read the note at line 1757; those keys are hardcoded in the sql files and I decided not to touch them at this time.